apps/daffio/src/app/docs/guide-docs/components/guides-list/guides-list.component.html

```diff
@@ -0,0 +1,9 @@
+<daff-accordion>
+	<daff-accordion-item>
+		<h2 daffAccordionItemTitle>{{ guideList.title }}</h2>
```

Comment:
I got rid of the root component and I moved `daff-accordion` into the sidebar. I removed the accordion from inside the guides-list component too because in the future (once I implement the nesting accordion stuff) there should only be one accordion (the accordion-items will derive their indent state from their parent accordion).